pylint/pyreverse/inspector.py

```diff
@@ -152,16 +152,6 @@ def visit_project(self, node: Project) -> None:
         for module in node.modules:
             self.visit(module)
 
-    def visit_package(self, node) -> None:
-        """visit an astroid.Package node
```

Comment:
I'm not too familiar with the `pyreverse` code, but even if it isn't called anymore, we shouldn't remove it here. That should be done in a separate PR.

Reply:
See #4952